dev/benchmarks/macrobenchmarks/pubspec.yaml

```diff
@@ -18,6 +18,8 @@ dependencies:
   #   flutter update-packages --force-upgrade
   flutter_gallery_assets: 1.0.2
 
+  url_launcher: 6.0.18
```

Comment:
Since this is pinned, this dependency will need to be updated (manually?) to start testing the optimized implementation.

Reply:
Semi-manually. A script does it for you, but you run the script manually and submit a PR.

Reply:
@yjbanov this is cool, we can gather several data points of the "old" dependency until it gets updated (by us or somebody else). The thing is that I don't need to wait to push the fix to the Link widget to gather data here!